lib/smart_answer_flows/marriage-abroad/outcomes/countries/azerbaijan/ceremony_country/partner_local/_opposite_sex.erb

```diff
@@ -0,0 +1,73 @@
+
+You’ll need to have been living in Azerbaijan for at least 3 full days before you can post notice, for example if you arrive in the country on Thursday, the earliest you can post notice is the following Monday.
+
+[Make an appointment at the embassy in Baku.](https://www.consular-appointments.service.gov.uk/fco/#!/british-embassy-baku/notice-of-marriage-or-civil-partnership/slot_picker)
+
+You’ll need to provide supporting documents, including:
+
+- your passport
+- proof of residence, such as a residence certificate - check with the embassy or notary public to find out what you need
+- equivalent documents for your partner
+
+You’ll need to complete a [‘Notice of marriage’](/government/publications/notice-of-marriage-form--2) form and an [‘Affidavit for marriage’](/government/publications/affirmationaffidavit-of-marital-status-form) form.
+
+You can download and fill in (but not sign) the forms in advance.
+
+You must take the forms with you to your appointment.
+
+%The names on all documents you provide must appear exactly as they do on your passports - if not, the authorities may refuse to allow the marriage to go ahead. You may need to provide evidence if the name on your passport is different to your birth certificate (for example marriage certificate or deed poll).%
+
+If you or your partner have been divorced, widowed or previously in a civil partnership, you’ll also need whichever of the following documents is appropriate:
+
+- a [decree absolute or final order](/copy-decree-absolute-final-order)
+- a civil partnership dissolution or annulment certificate
+- your (or your partner’s) former spouse or civil partner’s [death certificate](/order-copy-birth-death-marriage-certificate/)
+
+^You’ll also need to provide evidence of nationality or residence if the divorce or dissolution took place outside the UK. You’ll need to get it [legalised](/get-document-legalised) and [translated](/government/collections/lists-of-translators-and-interpreters) if it’s not in English.^
+
+###What happens next
+
+The embassy or notary public will charge a fee for taking the oath.
+
+If you give notice at a notary public, you must post the signed forms and any supporting documents to your nearest British embassy or consulate afterwards.
+
+^The embassy or consulate may charge a fee to return your documents to you.^
+
+The consulate will display your notice of marriage publicly for 7 days.
+
+They’ll then send all of your documentation to the British embassy or consulate nearest to where you’re getting married in Azerbaijan, who will issue your CNI (as long as nobody has registered an objection).
+
+There’s an additional fee for this - the embassy or consulate will contact you to arrange payment.
+
+You should check if your CNI needs to be legalised (certified as genuine) by the local authorities in Azerbaijan.
+
+^You don't need to stay in the country while your notice is posted.^
+
+##Naturalisation of your partner if they move to the UK
```

Comment:
@ikennaokpala I think this can be replaced with `<%= render partial: 'partner_naturalisation_in_uk' %>` in all files that contain this text